packages/block-template/package.json

```diff
@@ -46,6 +46,7 @@
     "core-js": "^2.6.12",
     "cross-env": "^7.0.3",
     "html-webpack-plugin": "^4.5.2",
+    "mock-block-dock": "0.0.2",
```

Comment:
We need to remember to bump `block-template` (possibly in this PR because we can't push changes to `main` directly)